content/en/monitors/notify/_index.md

```diff
@@ -56,7 +56,8 @@ Configure the renotify interval, the monitor states from which the monitor renot
 
 For example, configure the monitor to `stop renotifying after 1 occurrence` to receive a single escalation message after the main alert.
 
-If renotification is enabled, you are given the option to include an escalation message that is sent if the monitor remains in one of the chosen states for the specified time period.
+If renotification is enabled, you are given the option to include an escalation message that is sent if the monitor remains in one of the chosen states for the specified time period.  
+**Note:** Variables in the renotification are populated with the data available to the monitor in the time period of the renotification.
```

Comment:
I would suggest to move this one paragraph up, right below the `For example, configure ...` sentence.  Because the sentence here above is talking about how to define the escalation message and goes with the other paragraph below so it's like stopping a sentence midway if we were to keep this note here. 

I would also suggest to mention that it's attribute and tag variable that we are talking about and link to https://docs.datadoghq.com/monitors/notify/variables/?tab=is_alert#attribute-and-tag-variables